src/contexts/sendSearchRequest.ts

```diff
@@ -84,6 +86,9 @@ export const sendSearchRequest = async ({
           ? {
               reranking_config: {
                 reranker_id: rerankerId,
+                mmrConfig: {
+                  diversityBias: rerankDiversityBias,
```

Comment:
Why are `mmrConfig` and `diversityBias` camel-case whereas the other properties are snake-case?

Reply:
good point. I believe both work and that's why I didn't catch up, but will make consistent for the code to look better.